src/schemas/map.json

```diff
@@ -36,13 +36,10 @@
       "$ref": "state-machine.json#"
     },
     "Parameters": {
-      "type": ["object", "array"],
-      "items": {
-        "type": "object"
-      }
```

Comment:
not sure an array is allowed here

will need to check w/ the emulator/deployment. If it is, the items should be the payload template shape

Reply:
👌